app/kubemci/pkg/gcp/forwardingrule/interfaces.go

```diff
@@ -28,4 +28,6 @@ type ForwardingRuleSyncerInterface interface {
 	DeleteForwardingRules() error
 	// GetLoadBalancerStatus returns the struct describing the status of the given load balancer.
 	GetLoadBalancerStatus(lbName string) (*status.LoadBalancerStatus, error)
+	// ListForwardingRules returns all forwarding rules used by MCI.
+	ListForwardingRules() (string, error)
```

Comment:
Comparing it with GetLoadBalancerStatus, I see that this assumes that the result is to be printed and hence formats it for printing. GetLoadBalancerStatus, on the other hand, just returns structured data and then its upto the caller on how to use it.

Wondering if we should do the same here?
(am still thinking about it, so feel free to push back or discuss before making changes)